diff -Naur a/Makefile b/Makefile
--- a/Makefile	2020-06-04 12:27:42.000000000 +0000
+++ b/Makefile	2020-11-19 19:57:03.513481824 +0000
@@ -1,4 +1,4 @@
-PLUGIN_MODBUS:=0
+PLUGIN_MODBUS:=1
 BUILD:=debug
 
 SRC_DIR:=src
@@ -21,8 +21,7 @@
 VNC_OBJ:=$(addprefix $(BUILD_DIR)/, $(VNC_SRC:.c=.o))
 VNC_BIN:=vncproxy
 
-CPPFLAGS+=-I$(C8Y_LIB_PATH)/include $(shell pkg-config --cflags lua)\
-		  -DPKG_DIR='"$(PKG_DIR)"'
+CPPFLAGS+=-DPKG_DIR='"$(PKG_DIR)"'
 CXXFLAGS+=-Wall -pedantic -Wextra -std=c++11 -MMD
 LDFLAGS:=-Llib
 LDLIBS:=-lsera $(shell pkg-config --libs lua) -pthread
