diff -Naur cumulocity-agent-v4.2.8-orig/src/modbus/mbbase.h cumulocity-agent-v4.2.8/src/modbus/mbbase.h
--- cumulocity-agent-v4.2.8-orig/src/modbus/mbbase.h	2020-06-04 12:27:42.000000000 +0000
+++ cumulocity-agent-v4.2.8/src/modbus/mbbase.h	2020-12-08 18:33:02.326348595 +0000
@@ -140,6 +140,7 @@
                         + std::to_string(baud) + " " + par
                         + std::to_string(data) + ":" + std::to_string(stop));
         ctx = modbus_new_rtu(device, baud, par, data, stop);
+		modbus_rtu_set_serial_mode(ctx, MODBUS_RTU_RS485);
         init();
     }
 
@@ -151,6 +152,7 @@
                         + std::to_string(baud) + " " + par
                         + std::to_string(data) + ":" + std::to_string(stop));
         ctx = modbus_new_rtu(dev, baud, par, data, stop);
+		modbus_rtu_set_serial_mode(ctx, MODBUS_RTU_RS485);
         init();
     }
 };
@@ -168,4 +170,4 @@
     }
 };
 
-#endif /* MBBASE_H */
+#endif /* MBBASE_H */
\ No newline at end of file
